package.json

```diff
@@ -14,26 +14,28 @@
     "e2e:gui": "ENABLE_VUE_DEVTOOLS=true npx cypress open --config-file tests/cypress.config.js",
     "e2e:mockprod": "CUTTLE_ENV=production npm run e2e:server",
     "e2e:server": "VUE_APP_API_URL=http://localhost:1337 npx cypress run --config-file tests/cypress.config.js",
+    "e2e:server:gui": "VUE_APP_API_URL=http://localhost:1337 npx cypress open --config-file tests/cypress.config.js",
     "start": "node app.js",
     "start:client": "concurrently \"ENABLE_VUE_DEVTOOLS=true vue-cli-service serve\" \"npm run start:devtools\"",
     "start:dev": "concurrently \"npm run start:server\" \"npm run start:client\"",
     "start:devtools": "vue-devtools",
     "start:mockprod": "npm run build && CUTTLE_ENV=production npm run start:server",
+    "start:mockprod:tools": "concurrently \"NODE_ENV=dev ENABLE_VUE_DEVTOOLS=true npm run build && npm run start:server\" \"npm run start:devtools\"",
     "start:server": "node app.js"
   },
   "main": "app.js",
   "dependencies": {
     "connect-redis": "3.0.2",
-    "dayjs": "^1.11.3",
+    "dayjs": "^1.11.5",
     "lodash": "^4.17.21",
     "machinepack-passwords": "^2.3.0",
-    "sails": "^1.4.3",
+    "sails": "^1.5.3",
     "sails-disk": "^2.1.1",
     "sails-hook-orm": "^2.1.1",
     "sails-hook-sockets": "^1.5.5",
     "sails-postgresql": "^4.0.0",
     "sails.io.js": "^1.2.1",
-    "socket.io-client": "^2.3.0",
+    "socket.io-client": "^2.3.1",
```

Comment:
Updated a few deps in here but wanted to talk with you about this. Some of these are like ~7 years old at this point but a straight upgrade does break things. It also looks like the new sails generate adds some sailshq namespaced packages which threw me off.
```
"dependencies": {
  "sails": "^1.5.3",
  "sails-hook-apianalytics": "^2.0.3",
  "sails-hook-organics": "^2.0.0",
  "sails-hook-orm": "^4.0.0",
  "sails-hook-sockets": "^2.0.0",
  "@sailshq/connect-redis": "^3.2.1",
  "@sailshq/socket.io-redis": "^5.2.0",
  "@sailshq/lodash": "^3.10.3"
},
```

Reply:
I'll take a deeper look